90metalmdsquash/metal-md-lib.sh

```diff
@@ -294,16 +314,10 @@ add_sqfs() {
         fi
     fi
     mkdir -pv $sqfs_store
-    # if metal-nowipe=1, this will already exist
-    if mount -n -t xfs -L "${sqfs_drive_authority}" $sqfs_store; then
-        # nothing to do
-        umount $sqfs_store
-        return
-    elif mount -n -t xfs /dev/md/SQFS $sqfs_store; then
+    if mount -n -t xfs /dev/md/SQFS $sqfs_store; then
         mkdir -pv "$sqfs_store/$live_dir"
         fetch_sqfs "$sqfs_store/$live_dir" || metal_die 'Failed to fetch squashFS into squashFS storage!'
```

Comment:
Debating if this should be `metal_die -b` to instantly reboot, or if it should remain as-is to facilitate investigations.

Reply:
Probably not.. because I do want folks to be able to look and confirm that TCP/IP is broken.